CONTRIBUTING.md

```diff
@@ -289,6 +289,12 @@ from [the Shelley ledger spec](./eras/shelley/formal-spec)).
 4. Add a link to the package near the bottom of [flake.nix](./flake.nix),
    following the existing examples.
 
+### To build the Haskell code from the Agda ledger spec
```

Comment:
I would prefer this to be below the step that requires it, rather than at this level, since it doesn't seem like a "first class citizen" for things to do with the project.  But it's just a preference, if your change helps to make the document more readable and wieldy, feel free to ignore my suggestion.